spl-balance/src/lib.rs

```diff
@@ -0,0 +1,178 @@
+pub mod instruction;
+pub mod pb;
+
+use bs58;
+use pb::sol_token::v1 as proto;
+use substreams::log;
+use substreams::store::{StoreGet, StoreGetProto, StoreNew, StoreSet, StoreSetProto};
+use substreams_solana::pb::sol as solana;
+
+#[substreams::handlers::map]
```

Comment:
Perhaps you could move this to a modules folder like the other substreams are now doing?